open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/ffdc/SearchIntegratorAuditCode.java

```diff
@@ -40,8 +40,15 @@ public enum SearchIntegratorAuditCode implements AuditLogMessageSet {
             "The integration connector is running but does not have a context.  This is a timing issue in the integration daemon.",
             "Gather information about the connector's configuration, the types of metadata it was integrating, the audit log messages " +
                     "from the integration daemon and its partner metadata server.  Then contact the Egeria community to get help."),
-    ;
 
+    REGISTER_CATALOG_LISTENER("OMIS-SEARCH-INTEGRATOR-0004",
+            OMRSAuditLogRecordSeverity.ERROR,
+            "Integration connector {0} has an exception while attempting to register a Asset Catalog event listener",
+            "Resolve the issue preventing the registration of the listener",
```

Comment:
this is the system action, so does not seem appropriate. I am guessing the connector either continues without a context or is stopped. We should detail what the system is doing here

Reply:
I think this would be better served as an issue against the development of the connector, which I believe is still on going. That's s why the printStackTrace was there and the original fix suppressed the warnings.

Reply:
@wbittles I see 2 options , 
1)  we could amend the words of the system action, and change "register a " to register an". 
or
2) we could remove this part of the pr and raise a separate issue for this to be fixed by the code owner.  

I don't think we should knowingly add something that is incorrect.

Reply:
Apologises David I thought you were questioning the behaviour of the connector itself, I've went for option 1.

Reply:
Sorry Billy I do not think I am being clear. Could you also change the system action as well; changing  "Resolve the issue preventing the registration of the listener" to  "The integration connector is running, but does not have a listener". 
I just noticed that the name of it is REGISTER_CATALOG_LISTENER  it would be better to call it REGISTER_CATALOG_LISTENER_ERROR. 

Sorry for drip feeding you feedback in these reviews - I spotted more each time I looked at it.

Reply:
It's all good iterative development. 
I'm happy to make that change and I usually use _ERROR on the identifier but I had noticed that the message (that this message was modelled on) is called "NULL_CONTEXT" , it's the third Audit message.
Let me know if you still want the change and I'll add it ?

Reply:
as requested , I have just tested and pushed the commit

Reply:
Apologies for late comments. Agree with many of the comments. The code change here was needed to address the issue, but did then require a new audit message which might be best written by component author. 

I would be inclined to do as @wbittles did and propose an initial fix, tagging the component owner for review. Whilst the error may need changes (which have been resolved here...) I'd argue it is still an improvement over a stacktrace, which is our primary criteria for merging. But, If we then wanted to merge, and issues were still outstanding, I would raise a new issue to track that new event & it's refinement

Reply:
@planetf1 @davidradl This was only intended as an incremental fix over a call to stackTrace and with this code still under development I would expect changes imminently.

Reply:
@mandy-chessell as code owner, are you ok for this change to go in and then correct the message identifier in a later issue, or would you prefer it to be fixed here. We are talking of changing REGISTER_CATALOG_LISTENER to REGISTER_CATALOG_LISTENER_ERROR as the message is describing an error. This is in [open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/ffdc/SearchIntegratorAuditCode.java](open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/ffdc/SearchIntegratorAuditCode.java)

Reply:
@davidradl @mandy-chessell  We have also added a log message as opposed to the original fix which just suppressed the warnings.

Reply:
@davidradl It's not necessary to hold all this work up just because you have mistakenly asked for changed that you are now having second thoughts over.
This PR has passed all checks and can be merged while you and Mandy discuss the name of the variable in a separate issue ?

Reply:
It's useful to fix any obvious errors like typos - but I think our principles should be
a) Essential - does this code cause any regressions?
b) Essential - does this code improve on what we had before?
c) Preferable - is this code good enough for now with any obvious issues addressed directly, or via a new issue/intended fix soon

I think this code passed a,b pretty much straight away, and thanks to reviews gets to c. it's often easiest/best to fix what is obvious within the pr, but not essential if we raise the issue to do later

It's not perfect (ie its->it's !) but I would approve merging. Since the component owner may be best placed to review the text of the message, and is active in this area I would raise another issue for this to be clarified - and also for CODEOWNERS to be updated with the correct owner, but that can be dealt with asynchronously.